echoregions/regions2d/regions2d.py

```diff
@@ -1,59 +1,52 @@
 from pathlib import Path
 from typing import Dict, Iterable, List, Union
 
+import matplotlib
+import matplotlib.pyplot as plt
 import numpy as np
 from numpy import ndarray
 from pandas import DataFrame, Series
-from xarray import DataArray
 
-from ..convert import utils
-from ..convert.evr_parser import Regions2DParser
-from ..plot.region_plot import Regions2DPlotter
-from . import Geometry
+from ..utils.io import validate_path
+from ..utils.time import parse_simrad_fname_time
+from .regions2d_parser import parse_regions_file
 
 
-class Regions2D(Geometry):
+class Regions2D:
     def __init__(
         self,
-        input_file: str = None,
-        parse: bool = True,
-        offset: Union[int, float] = 0,
+        input_file: str,
         min_depth: Union[int, float] = None,
         max_depth: Union[int, float] = None,
         depth: ndarray = None,
     ):
-        super().__init__()
-        self._parser = Regions2DParser(input_file)
-        self._plotter = None
-        self._masker = None
+        self.depth = (
+            None  # Single array that can be used to obtain min_depth and max_depth
+        )
+        self._min_depth = (
+            None  # Set to replace -9999.99 depth values which are EVR min range
+        )
+        self._max_depth = (
+            None  # Set to replace 9999.99 depth values which are EVR max range
+        )
```

Comment:
I think we can remove `self.depth` to not have a "hidden" thing for setting the max and min depth. Similar to the case in Lines, I also wonder if we can simply keep min_depth and max_depth as attributes and not properties to make things simpler, since it is unlikely these values will be set by some complicated method.